wrangler-transform/pom.xml

```diff
@@ -84,6 +84,12 @@
         </exclusion>
       </exclusions>
     </dependency>
+    <dependency>
+      <groupId>io.cdap.wrangler</groupId>
+      <artifactId>wrangler-storage</artifactId>
+      <version>4.12.0-SNAPSHOT</version>
+      <scope>compile</scope>
+    </dependency>
```

Comment:
It is not recommended to add this dependency in wrangler-transform.

Please remove it.